src/internal/HtmlColorParse.cpp

```diff
@@ -0,0 +1,54 @@
+/*-------------------------------------------------------------------------
+HtmlColor::Parse() implementation
+
+Written by Unai Uribarri
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+#include "HtmlColor.h"
+
+bool HtmlColor::Parse(const char* name)
```

Comment:
If you move this into the HtmlColorNames.cpp file, then you could get rid of the null terminating list as then you could use sizeof()/sizeof([]).  Again, this library is aimed at some small embedded procs with very little code memory; reducing where we can is important.